testgoogleapi.py

```diff
@@ -0,0 +1,27 @@
+from typing import List
+from googleapiclient.discovery import build
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+
+
+my_api_key = os.getenv("GOOGLE_API_KEY")
+my_cse_id = os.getenv("GOOGLE_CSE_ID")
```

Comment:
How do I get those? Can you send me a link

Reply:
I added instructions in the README.md, feel free to reach out if you need